docs/source/community/troubleshooting.rst

```diff
@@ -86,7 +85,7 @@ ERROR - Failed to publish, wsi: ..., tsi: XXXXX
 -----------------------------------------------
 
 Data arrived for a station that is not present in the station metadata cache. 
-To add missing stations, update the file ``metadata/station/station_list.csv`` in the wis2box data directory (see :ref:`setup`).
+To add missing stations, use the station-editor in wis2box-webapp (from wis2box-1.0b5) or update the file ``metadata/station/station_list.csv`` in the wis2box data directory (see :ref:`setup`).
```

Comment:
Does updating station_list still update the station collection? I know we had a discussion on this but can't remember what the outcome was.

Reply:
You can still use the station_list.csv to do a bulk-insert the station collection, by executing the command 'wis2box metadata station publish-collection', as documented in setup.rst.